frameworks/PHP/laravel/deploy/roadrunner/.rr.yaml

```diff
@@ -7,16 +10,14 @@ logs:
   level: error
 http:
   address: 0.0.0.0:8080
-  middleware: ["headers", "static", "gzip"]
+  middleware: ["headers"]
```

Comment:
@joanhey 👋 
You don't need to use middleware at all here. You may safely remove the whole `middleware` line + headers section.

Reply:
If I remove it, don't pass the tests.
The test need a server: header